ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -22,6 +22,10 @@ def solve(x, y):
     return ivy.solve(x, y)
 
 
+def logdet(matrix, name=None):
+    return ivy.det(matrix).log()
+
+
```

Comment:
`supported_dtypes` should be added here as `logdet()` only accepts matrices of specific dtypes according to the tensorflow documentation.
`complex dtypes` are not to be considered while adding, as we are not yet testing for them, only consider the `float` ones.